vendor/assets/javascripts/jquery.textile.js

```diff
@@ -192,8 +192,10 @@
       $.post(this.$element.data('preview-url'),
         data,
         function(result) {
-          this.options.$preview.removeClass('loading-indicator')
-            .html(result);
+          this.options.$preview.removeClass('loading-indicator').html(result);
+          if (result == '\n') {
```

Comment:
Can the result actually be empty string and not just a new line?

We've also got to think about new line variations. `\r` vs `\n` vs `\r\n`.

Reply:
I take it back. `\n` should be fine. It looks like the server side rendering is creating the new line which I think will mean it always results in a unix newline. So just checking \n should be fine. If someone on a windows machine wants to feed in a blank `\r` and have it rendering nothing that's fine with me.